terminus/geometry/bounding_box.py

```diff
@@ -0,0 +1,33 @@
+from geometry.point import Point
+
+
+class BoundingBox(object):
+
+    def _normalize(self):
+        new_origin = Point(min(self.origin.x, self.corner.x),
+                           min(self.origin.y, self.corner.y))
+        new_corner = Point(max(self.origin.x, self.corner.x),
+                           max(self.origin.y, self.corner.y))
+        self.origin = new_origin
+        self.corner = new_corner
+
+    def __init__(self, origin, corner):
+        self.origin = origin
+        self.corner = corner
+        self._normalize()
+
+    def __eq__(self, other):
```

Comment:
+1 for redefining the equality! Please ping me when you have a moment so we can briefly discuss the `__hash__` implementation you must also add when redefining the equality.